tests/compiler.js

```
@@ -694,6 +694,38 @@
                   { name: 'thedude', data: {tmpl: 'include.html'} },
                   'hello world FooInclude thedude');
 
+            equal('hello world {% include "missing.html" ignore missing %}',
+                  'hello world ');
+
+            equal('hello world {% include "missing.html" ignore missing %}',
+                  { name: 'thedude' },
+                  'hello world ');
+
+            finish(done);
+        });
+
+        it('should throw an error when including a file that does not exist', function(done) {
+            render(
+                '{% include "missing.html" %}',
+                {},
+                { noThrow: true },
+                function(err, res) {
+                    expect(res).to.be(undefined);
+                    expect(err).to.match(/template not found: missing.html/);
+                }
+            );
+
+            finish(done);
+        });
+
+        it('should include templates that does not exist if the error is suppressed', function(done) {
```

The wording here also doesn't seem quite right -- it's not possible to "include templates that does not exist." I would say instead something like `it('should fail silently on missing templates if requested')`